e2e/tests/user/registration.spec.ts

```diff
@@ -0,0 +1,122 @@
+import {ElementHandle} from 'puppeteer';
+import CreateAccountPage from '../../app/create-account-page';
+import GoogleLoginPage from '../../app/google-login';
+import PuppeteerLaunch from '../../services/puppeteer-launch';
+require('../../driver/waitFuncs');
```

Comment:
I'm curious, why do you require(...) in some cases, and in some other cases use "import foo from ..."?

In most of our React codebase we use module imports, except for a few cases in test code where we use require(...). Do you know of a significant difference between them that guides you to use one versus the other?

Reply:
`require` and `import` are used due to my inability to convert all `require` to `import`. I had  difficult time to make conversion works. I can try convert all `require` in a following PR.